packages/plugin-ui/src/lib/plugin-extension/extension-registry.service.ts

```
@@ -0,0 +1,115 @@
+				this._pluginToExtensions.set(options.pluginId, entries);
+			}
+		}
+	}
+
+	/**
+	 * Registers multiple extensions.
+	 * @param extensions Array of extension definitions.
+	 * @param options Optional pluginId for lifecycle cleanup.
+	 */
+	registerAll(extensions: ExtensionDefinition[], options?: ExtensionRegistrationOptions): void {
+		extensions.forEach((e) => this.register(e, options));
```

Biome (2.3.14)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)